templates/taxi/car_detail.html

```diff
@@ -9,4 +9,6 @@ <h1>Drivers</h1>
     <li>{{ driver.username }} ({{ driver.first_name }} {{ driver.last_name }})</li>
   {% endfor %}
   </ul>
+  <a href="{% url 'taxi:car-update' pk=car.id %}" class="btn btn-warning">Update car</a>
+  <a href="{% url 'taxi:car-delete' pk=car.id %}" class="btn btn-warning">Delete</a>
```

Comment:
The 'Update car' and 'Delete' buttons should be moved outside the loop iterating over drivers to ensure they are displayed only once. Currently, they will repeat for each driver listed.